jsparagus/actions.py

```diff
@@ -100,6 +100,12 @@ def __lt__(self, other):
     def __repr__(self):
         return str(self)
 
+    def __str__(self):
+        return repr(self)
```

Comment:
This seems like it can't be right -- `__str__` calling `repr` and `__repr __`calling `str`.